R/create_game_state.R

```diff
@@ -0,0 +1,23 @@
+#' Create a game state.
+#'
+#' @param a An arena.
+#' @param p A player.
+#'
+#' @return A game state.
+#' @export
+#'
+#' @examples
+#' arena <- create_arena()
+#' player <- list()
+#' player$x <- 100.0
+#' player$y <- 200.0
+#' player$size <- 1
+#' game_state <- create_game_state(arena, player)
+create_game_state <- function(a, p) {
+  if (!is.character(a)) stop("The first argument must be a character vector.")
+  if (!is.list(p)) stop("The second argument must be a list.")
+  l <- list()
+  l$arena <- a
+  l$player <- p
+  return(l)
```

Comment:
Tip: you can just wrote `l` here (the `return` is redundant when it is the last statement of a function)